Mod0/TechnicalDay2.md

```diff
@@ -0,0 +1,38 @@
+
+We also use non-tangible objects like vocabulary, numbers, and dates which allow us to communicate with each other; let's call these objects `data`.  We can use data to communicate our needs, perform math and statistics, and relate spans of time (and many other tasks!). Using data, we can understand sentences like: On Tuesday, my son will be 12 years old. He is getting 2 lego sets for his birthday.
+
+With objects and data, we have everything we need to accomplish our daily tasks.  But there is one more concept that humans rely on: lists.  We use lists to help organize all of the objects and data we know about.  Let's imagine we are moving into a new home.  We know that our clothes and furniture need to be moved.  From this, we can infer that multiple pieces of clothing and multiple pieces of furniture exist and must be moved; we use the idea of `lists` instead of explicitly stating every single item that needs to be moved.
+
+Lists also help us accomplish similar tasks more efficiently.  Let's say I need groceries.  I _could_ go to the grocery store, pick up milk, come home, put the milk away, go to the grocery store, pick up bread, come home, put the bread away, go to the gocery store, pick up coffee, come home, put the coffee away, etc...  While this method would work, it is not very efficient; it is much better to create a list of all the items I need from the grocery store, then use that list at the store to pick up everything I need.
+
+
+### Things and Computers
+
+Unlike humans, computers don't know about very many things or data; but, they do know a few types of things and a few types of data. And, they can store lists!
+
+Work through the tutorial linked below to learn about the things and data that c# knows.  Make sure to walk through each `Try it yourself` section.
+* [Built in Types](https://www.w3schools.com/cs/cs_data_types.php)
+
+Work through the tutorials below to learn about how c# manages lists, and uses lists to accomplish specific tasks. Make sure to walk through each `Try it yourself` section.
+* [Arrays](https://www.w3schools.com/cs/cs_arrays.php)
+* [Looping](https://www.w3schools.com/cs/cs_arrays_loop.php)
+
+## Putting it all together
+
+Open [this repl](https://replit.com/@MeganMcMahon1/WorkingWithLists#main.cs) and complete the following:
+1. Fork the repl so that you have your own copy.
+2. Identify as many of the tools you learned about above as you can.  Make a [comment](https://www.w3schools.com/cs/cs_comments.php) above each line where you see something you recognize. You may not recognize everything in this code - that's ok!
+3. Run the program by clicking on the green play button!
+
+## Reflection Questions
+In a notebook, reflect on the following:
+1. Look around you and see if you can group the objects that you see into one or more lists.  Ex: games, books.
+2. Imagine you are going to create an application like Instagram; what types of objects will you program need to know about?
```

Comment:
I really like this bringing the topic of bringing in a real world application here! I wonder if instead of question 1, we also make that about Instagram? Can you think of any ways those objects might be grouped into lists?

Could also have folks think about the data that exists in an app like instagram. Things like username, number of posts, etc. Maybe even put in an example image of instagram to help spark ideas. https://jessicathiefels.com/wp-content/uploads/2021/04/Screen-Shot-2021-04-20-at-2.34.49-PM-1-1024x1169.png

Just brainstorming here, feel free to take however much or however little. I'm just liking the idea of thinking about how all the things we've been talking about fit into the real world.